src/modules/schema.js

```diff
@@ -13,6 +14,7 @@ export const BLOB_ID_MAX_BYTE_SIZE = 32;
 // implementer may choose to implement this as a dynamic config option settable with the `parameter_type!` macro
 export const BLOB_MAX_BYTE_SIZE = 1024;
 
+// TODO: Remove
 // Schema-validator spec
 const jsonSchemaSpec = {
```

Comment:
happy to remove this, was purely for testing w/o http because was unsure if request should be here or somewhere else (since method of validateSchema was never marked async), thought maybe it could pass the schema obj somehow